MANIFEST.in

```diff
@@ -6,3 +6,6 @@ include setup.py
 include tox.ini
 graft doc
 graft tests
+
+global-exclude __pycache__
```

Comment:
still unsure why we need these includes?

Reply:
They are excludes :) because otherwise it is possible to pollute a release package with these files.

Reply:
see my point in https://github.com/tox-dev/tox/issues/754#issuecomment-385339365